aws/scripts/sh/create_eventbridge_rule.sh

```
@@ -50,7 +50,14 @@ fi
 
 account_id=$(aws sts get-caller-identity --query "Account" --output text)
 
-# Add target (Lambda) to the rule
+# 💣 Clean up old targets first to avoid hitting the limit
+existing_targets=$(aws events list-targets-by-rule --rule "$rule_name" --query "Targets[].Id" --output text)
+if [ -n "$existing_targets" ]; then
+  echo "Removing existing targets to avoid hitting target limit..."
+  aws events remove-targets --rule "$rule_name" --ids $existing_targets
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)